swift/Lottie.swift

```diff
@@ -17,15 +17,6 @@ public enum LottieError: Error {
 /// Shorthand for the buffer type used to represent image pixel data.
 public typealias Buffer = UnsafeMutablePointer<UInt32>
 
-/// Shorthand for a pointer to an internal Lottie animation object
-private typealias Animation = OpaquePointer
-
-/// Shorthand for a pointer to an internal rendering canvas
-private typealias Canvas = OpaquePointer
-
-/// Shorthand for a pointer to an internal picture object
-private typealias Picture = OpaquePointer
-
```

Comment:
I've had to remove these for the time being as their names are clashing with the new internal types.